scripts/index-demo-page.js

```diff
@@ -256,7 +256,7 @@
           liveui: stateEls.liveui.checked,
           html5: {
             hls: {
-              overrideNative: !window.videojs.browser.IS_SAFARI
+              overrideNative: true
```

Comment:
This code change is only really testable on new edge and safari as other browsers don't support many native types.